srcpkgs/mdr/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'mdr'
+pkgname=mdr
+version=0.2.4
+revision=1
+build_style=go
+go_import_path="github.com/MichaelMure/mdr"
+go_get="no"
```

Comment:
I believe this is the default.

Reply:
I'm not sure about what I should change.

Reply:
Drop this line